README.md

```diff
@@ -22,4 +22,39 @@ document](CONTRIBUTING.md).
 Issues can be filled in our [GitHub issue
 tracker](https://github.com/input-output-hk/ouroboros-consensus/issues).
 
+## Installation
```

Comment:
Perhaps this should be a sub-section of `Documentation`: once we add the Consensus code this could lead to confusion.

Reply:
We shall rewrite the `README` when we merge consensus code in this repo, I guess/